### PR TITLE
Fix repo org move issues

### PR DESCRIPTION
**Pull request by jdisanti**
This PR fixes issues introduced by moving the repository from awslabs/smithy-rs to smithy-lang/smithy-rs.

----

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
-/rust-runtime/aws-smithy-http-server-python/        @awslabs/smithy-rs-python-server @awslabs/smithy-rs-server
+/codegen-server-test/python/                        @smithy-lang/smithy-rs-python-server @smithy-lang/smithy-rs-server
+/codegen-server/python/                             @smithy-lang/smithy-rs-python-server @smithy-lang/smithy-rs-server
+/rust-runtime/aws-smithy-http-server-python/        @smithy-lang/smithy-rs-python-server @smithy-lang/smithy-rs-server
 
 # Typescript Server
-/codegen-server-test/typescript/                    @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
-/codegen-server/typescript/                         @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
-/rust-runtime/aws-smithy-http-server-typescript/    @awslabs/smithy-rs-typescript-server @awslabs/smithy-rs-server
+/codegen-server-test/typescript/                    @smithy-lang/smithy-rs-typescript-server @smithy-lang/smithy-rs-server
```
Comment (ysaito1001) [inline CODEOWNERS:14]: @david-perez, @drganjoo 
I don't see [smithy-rs-python-server](https://github.com/orgs/smithy-lang/teams/?query=smithy-rs-python-server) and [smithy-rs-typescript-server](https://github.com/orgs/smithy-lang/teams/?query=smithy-rs-typescript-server) under `smithy-lang` but do see them under [awslabs](https://github.com/orgs/awslabs/teams). Do they need to be moved to `smithy-lang`?

File: CODEOWNERS [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
+/codegen-server/typescript/                         @smithy-lang/smithy-rs-typescript-server @smithy-lang/smithy-rs-server
+/rust-runtime/aws-smithy-http-server-typescript/    @smithy-lang/smithy-rs-typescript-server @smithy-lang/smithy-rs-server
```
Comment (ysaito1001) [inline CODEOWNERS:16]: @drganjoo, @david-perez 
I don't see `smithy-rs-typescript-server` or `smithy-rs-python-server` under [smithy-lang](https://github.com/orgs/smithy-lang/teams/) but do see them under [awslabs](https://github.com/orgs/awslabs/teams). Do they need to be moved to `smithy-lang`?

Comment (rcoh) [reply CODEOWNERS:16]: if we want them to continue existing, we do. Not clear on that, cc @drganjoo

Comment (jdisanti) [reply CODEOWNERS:16]: Talked to @drganjoo. We'll change these ones to `smithy-rs-server`.

**Review by drganjoo — APPROVED** [review]

**Review by ysaito1001 — APPROVED** [review]

**Pull request merged**